go/go.mod

```diff
@@ -0,0 +1,34 @@
+module breakfast/app
```

Comment:
Instead of breakfast, should this be something like `vacation`? Also, I think go convention is to name modules based on their github URLs, e.g., `module github.com/efortuna/sagas-temporal-trip-booking`. Probably low-priority if it's intended to never be shared code (as in, imported by others).

Reply:
doh! yes.